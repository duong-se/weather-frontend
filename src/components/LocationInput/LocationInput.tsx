import React from 'react'
import { ReactComponent as SearchIcon } from './SearchIcon.svg'

export const TextInput: React.FC = () => {
  return (
    <div className="input-group flex-nowrap">
      <div className="input-group-prepend">
        <span className="input-group-text"><SearchIcon/></span>
      </div>
      <input name="locationName" type="text" className="form-control" id="locationName" placeholder="Search" />
    </div>
  )
}

export default TextInput
