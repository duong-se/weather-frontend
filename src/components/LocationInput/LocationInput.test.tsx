import React from 'react'
import { shallow } from "enzyme"
import LocationInput from "./LocationInput"

describe('LocationInput', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<LocationInput />)
    expect(wrapper).toMatchSnapshot()
  })
})
