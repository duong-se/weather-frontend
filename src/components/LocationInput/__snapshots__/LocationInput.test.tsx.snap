// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocationInput should match snapshot 1`] = `
<div
  className="input-group flex-nowrap"
>
  <div
    className="input-group-prepend"
  >
    <span
      className="input-group-text"
    >
      <ForwardRef(SvgSearchIcon) />
    </span>
  </div>
  <input
    className="form-control"
    id="locationName"
    name="locationName"
    placeholder="Search"
    required={true}
    type="text"
  />
</div>
`;
